src/test/unit/views/list-download-files.test.ts

```diff
@@ -0,0 +1,57 @@
+import {app} from '../../../main/app';
```

Comment:
Not this file, but I think we should add SJP view tests too for the new functionality